dm/config/testdata/ca.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
How about generate this test files in the test code instead of put this dummy files in our source code